src/humancellatlas/data/metadata/helpers/schema_validation.py

```diff
@@ -0,0 +1,42 @@
+from functools import (
+    lru_cache,
+)
+import json
+import logging
+
+from jsonschema import (
+    FormatChecker,
+    ValidationError,
+    validate,
+)
+import requests
+
+from humancellatlas.data.metadata.api import (
+    JSON,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class SchemaValidator:
+
+    @classmethod
+    def validate_json(cls, file_json: JSON, file_name: str):
+        # logger.debug('Validating JSON of %s', file_name)
+        try:
+            schema = cls._download_schema(file_json['describedBy'])
+        except json.decoder.JSONDecodeError as e:
+            raise Exception(f"Unable to parse json from {file_json['describedBy']} for file {file_name}") from e
```

Comment:
LTL

Extract variable, use additional argument to exception constructor instead of f string whenever possible.